english/dogrelax/dogrelax.md

```diff
@@ -0,0 +1 @@
+You notice she is chewing on a pink sock. You decide to [check out the sock](fill-sock.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.